packages/commands/generate/src/index.ts

```diff
@@ -29,14 +25,8 @@ export default defineCommand<{}, CliFlags>(() => {
         },
       });
     },
-    async handler(args) {
-      if (args.backend) {
-        await generateUsingPlugins(args);
```

Comment:
We have now 2 concepts of plugins: Plugins that transform schema at runtime and plugins that affect filesystem - like client side queries